ts/internal/runner.ts

```diff
@@ -155,6 +155,11 @@ const findExportables = (
   for (const [name, value] of Object.entries(config)) {
     if (isProject(value)) {
       result[name] = value;
+      const nested = findExportables(value);
+      for (const key in nested) {
+        if (key === "defaultExecutable") continue;
+        result[`${name}/${key}`] = nested[key];
+      }
```

Comment:
What will this code do for `Project`s within `Project`s? I'm not sure it'll do something sensible. So maybe filter out `Project`s here? At least until we have tests for nesting `Project`s?

Reply:
I haven't tried it, but I would imagine it would properly nest as expected, no? `findExportables` currently just creats a map from string to Projects/executables.

So in that case a project's keys would be concatenated here, right?

Reply:
Added a test for this, but to make it pass we needed to add `Project` to `Nestable`. I think this might be the right time to add it, but also if we're not ready to add it yet then I think maybe it's fine to leave this as is without the project check since typescript will prevent nested projects from occurring anyway

Reply:
Yeah, I think it's fine to have nested Executables now.